### DISCUSSION TITLE
DietPi-Software | Ring programming language

### DISCUSSION THREAD

**Discussion by faustf** [MichaIng/DietPi#6109] [Software Request]
#### Formal software information
- Software title | ring language program
- Short description | Simple and flexible programming language for applications development
- Official URL (if available) | https://ring-lang.github.io/
- GitHub URL (if available) |  https://github.com/MahmoudFayed
- (Official) install documentation | readme inside of packet
#### Are there similar/alternative software titles available with DietPi-Software?

- ...

#### What makes your requested software better than the above solutions, if available?
simply to use, multi OS , can compile or use like script , you can use natural language for programming , and crete extention in C or C++ very fast 
- ...

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
i dont know 
- ...

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

1. ...

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.

- ...

**Comment by MichaIng** [comment]
Migrated to discussion, so people can vote via ⬆️ button at above post.
